tests/nlu/utils/test_hf_transformers.py

```diff
@@ -132,330 +132,3 @@ def test_attention_mask(
 
     assert np.all(mask_ones == 1)
     assert np.all(mask_zeros == 0)
-
-
-# TODO: need to fix this failing test
-@pytest.mark.skip(reason="Results in random crashing of github action workers")
```

Comment:
Correct me if I am wrong but this test shouldn't actually be removed but decorated with `skip_on_ci` decorated and refactored to match the test cases you have constructed for `test_lm_featurizer_shape_values` inside `test_lm_featurizer.py`.